protocol/src/main/scala/io/mediachain/protocol/Datastore.scala

```diff
@@ -95,12 +101,6 @@ object Datastore {
       meta.get("type")
```

Comment:
ok, got rid of it altogether -- all subtypes correctly define their type appropriately and the user has no business specifying the type anyway.